spring-aop/src/main/java/org/springframework/aop/aspectj/AbstractAspectJAdvice.java

```diff
@@ -268,18 +268,16 @@ public void setArgumentNamesFromStringArray(String... args) {
 						this.argumentNames[i] + "' that is not a valid Java identifier");
 			}
 		}
-		if (this.argumentNames != null) {
```

Comment:
We should check it
`@Nullable
	private String[] argumentNames;`

Reply:
It looks like it cannot be null, it is always a string array